MapboxNavigation/MGLAccountManager.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import Mapbox
+
+
+extension MGLAccountManager{
+    
+    static let mapboxChinaBaseAPIURL = "https://api.mapbox.cn"
+    static let mapboxChinaBaseURLHost = "api.mapbox.cn"
+    static let mapboxChinaDayStyleURL = "mapbox://styles/mapbox/streets-zh-v1"
+    static let mapboxChinaNightStyleURL = "mapbox://styles/mapbox/dark-zh-v1"
+    
+    // Value of whether the map is China map or not
+    public class var hasChinaBaseURL : Bool{
```

Comment:
Since this is public, let’s make it available to Objective-C code using `@objc`.